### PR TITLE
Modal: Add <li> in OptionsBox

### PR DESCRIPTION
**Pull request by kaiofelipejs**
Options Box list does not contain `<li>` and this is a tech debt because screen readers and other assistive technologies depend on lists being structured properly to keep users informed of content within the lists.

This PR fixes this by adding a li (named `OptionItem`) in OptionsBox.

Reference: https://web.dev/list/

_Obs: If this PR is able to participate in Hacktoberfest, add the label `hacktoberfest-accepted` to count in my contributions :)_

### CODE REVIEW

File: packages/core/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -261,6 +269,7 @@ const ModalRenderer = (
                   key={name}
                   onMouseDown={() => setMenuOpened(name)}
                   active={active}
+                  data-testid="menu-item"
```
Comment (ggdaltoso) [inline packages/core/components/Modal/Modal.tsx:272]: Adding this `data-testid` attribute will ship it with `@react95/core` package, meaning that if someone uses this component somewhere in their application and has a test for it, they could not use this `menu-item` to test your own application. Also, despite the unity test, there is no need to have this property set on the React95 bundle prod.

Don't you agree?

**Review by ggdaltoso — CHANGES_REQUESTED** [review]
Thanks for your contribution, @kaiofelipejs !

If I remember well, I saw this issue on the Lighthouse report.

**Comment by allyssonsantos** [comment]
@all-contributors please add @kaiofelipejs for code, doc, and test

File: packages/core/components/Modal/Modal.tsx (hunk 1 above)
Comment (kaiofelipejs) [reply packages/core/components/Modal/Modal.tsx:272]: Yep, make sense. 

What you think about change the assertion to: 

```javascript
expect(getByText('Menu Text')).toBeInTheDocument();
```

This way we can remove `data-testid` attribute.

Comment (kaiofelipejs) [reply packages/core/components/Modal/Modal.tsx]: Done [here](https://github.com/React95/React95/pull/184/commits/13bfefbce8c2dcb41003e2aa1542357f3ffbea42).

**Review by ggdaltoso — APPROVED** [review]

**Pull request merged**